backend/person-service/src/main/kotlin/com/nosql/personservice/entity/ProjectEntity.kt

```diff
@@ -0,0 +1,16 @@
+package com.nosql.personservice.entity
+
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+import org.springframework.data.mongodb.core.mapping.Field
+
+@Document("project_document")
```

Comment:
```suggestion
@Document("project")
```

and check other entities

Reply:
fixed